docroot/profiles/custom/cgov_site/modules/custom/pdq_cancer_information_summary/src/Plugin/rest/resource/PDQResource.php

```diff
@@ -160,21 +162,20 @@ public function get($id) {
         }
         $fields[$code] = [
           'title' => $translation->getTitle(),
-          'cdr_id' => $translation->field_pdq_cdr_id->value,
-          'audience' => $translation->field_pdq_audience->value,
-          'summary_type' => $translation->field_pdq_summary_type->value,
-          'posted_date' => $translation->field_date_posted->value,
-          'updated_date' => $translation->field_date_updated->value,
-          'browser_title' => $translation->field_browser_title->value,
-          'cthp_card_title' => $translation->field_cthp_card_title->value,
```

Comment:
Why is cthp_card_title being removed??? This is the PDQ API and you are changing logic here.

Reply:
I'm seeing field_cthp_card_title in the field list in my local after reinstalling, i have reverted the changes i made.